spec/lcm/integration/lcm_end_to_end_spec.rb

```diff
@@ -63,6 +63,15 @@
       let(:schedule_additional_hidden_params) { schedule_additional_hidden_params }
       let(:output_stage_prefix) { output_stage_prefix }
     end
+
+    it 'does not transfer ADD components' do
```

Comment:
Why is this behaviour specified only for release brick? Shouldn't every synchronization brick behave this way?

Reply:
We could run this example for every brick but since the `ADD component` is not transferred to the master project, the test would be doing nothing for `provisioning` and `rollout`. And it takes seconds at least.

On the other side, we should run it for both runs of the release brick.